packages/shared-components/src/Forms/FillInFormPage.jsx

```diff
@@ -38,10 +38,9 @@ export const FillInFormPage = ({ form, submission, setSubmission, formUrl }) =>
     }
   }
 
-  function onNextOrPreviousPage({ page }) {
```

Comment:
Legg gjerne til Cypress-test for denne logikken.
Se om det er mulig å fjerne if-en.